application/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
+
+STATUS=0
+
+case "${MODE}" in
+    backup|restore)
```

Comment:
Should I see a left parenthesis somewhere?

Reply:
According to the man page it is optional.  I didn't know that until you pointed it out.  I've always seen shell script 'case' statements without the left parenthesis and just followed the pattern.

     The syntax of the case command is

           case word in
           [(]pattern) list ;;
           ...
           esac

     The pattern can actually be one or more patterns (see Shell Patterns
     described later), separated by “|” characters.  The “(” character before
     the pattern is optional.